### PR TITLE
:robot: Add garbageman.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/343 :rocket:

### CODE REVIEW

File: packages/garbageman.vm/garbageman.vm.nuspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>garbageman.vm</id>
+    <version>0.2.4</version>
+    <authors>alphillips-lab</authors>
+    <description>A set of tools designed for .NET heap analysis. These tools offer the following benefits for malware researchers: Ability to extract clear-text; payload (PE Images etc.) from .NET heaps quickly; Easy analysis of encrypted network protocols, signs of data exfiltration, and similar; Ability to overcome malware anti-dumping techniques (psnotify)</description>
```
Comment (Ana06) [inline packages/garbageman.vm/garbageman.vm.nuspec:7]: I would short this description to:
```suggestion
    <description>A set of tools designed for .NET heap analysis.</description>
```


From [our wiki](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#description):
> The description should be short and not include version specific details or other information that is likely to change in a future version.

**Review by Ana06 — CHANGES_REQUESTED** [review]
Similarly as with https://github.com/mandiant/VM-Packages/issues/368, we are missing dependencies here.

**Comment by emtuls** [comment]
There is an issue with `GarbageMan` in that it will display an error when opening the GUI, due to relative path usage of a file it depends on. 

The tool works properly if you open it directly from its extracted position, which is not ideal, but could work for the meantime if we intend to merge this before a fix is made.

I have opened an issue for the project repo in regards to the error: https://github.com/WithSecureLabs/GarbageMan/issues/5

image

**Review by emtuls — APPROVED** [review]

**Review by Ana06 — COMMENTED** [review]
I think that is a ugly error and that we should figure out a solution before getting this merged. What about using the current directory the path of GarbageMan? We may not be able to use the helper function or we may need to modify the helper function to add this option.

**Pull request merged**